---
layout: default
---
<div class="home">
  <div class="materials-wrap">




        
    <h2 class="module-header">1/31/2021 Announcement: First day of classes: Welcome to Spring 2022!</h2>
    <p>6.884 students: our the first round of sign ups for paper presentations covering September presentation slots are now up. View it <a href="/assets/6884PaperPresentationSlots.pdf">here</a>!
    </p>
      
      
    <p>Additional course websites:</p>
    <ul>
        <li><a href="https://stellar.mit.edu/S/course/6/fa19/6.884/index.html">MIT Stellar</a></li>
        <li><a href="https://piazza.com/class/jzvee3jyo6z28i">Piazza</a> (discussion forum)</li>
        <li><a href="https://learning-modules.mit.edu/gradebook/index.html?uuid=/course/6/fa19/6.884#assignments">Learning Modules</a> ((where you download problem set assignments and upload your solutions)</li>
    </ul>

    <h2 class="module-header">Course description</h2>
    <p>Welcome to an exploration of computational challenges in the design of human therapeutics.  We will explore computational methods in the design and analysis of therapeutics, including small molecules, biologics, cell based therapies, and synthetic biology approaches.   Lectures will present essential computational methods on molecular design and optimization drawing upon recent results in machine learning.  Classes will include presentations by students on recent research results related to the computational design of therapeutics and efficacy.   Problem sets will explore computational methods for therapeutic design.  Topics include protein design, antibody optimization, small molecule design and characterization, and the engineering of viruses and cell lines for therapeutic effect. Experts from industry will present on their views of the promise of computational approaches, what is working, and what is needed.
    </p>

    <p>As part of the subject students taking the graduate version  will use cloud resources to implement solutions to problems in therapeutic design, first in problem sets that span a carefully chosen set of tasks, and then in an independent project. You will be programming using Python 3 and TensorFlow 1.12 in Jupyter Notebooks on the Google Cloud, a nod to the importance of carefully documenting your work so it can be precisely reproduced by others.</p>


    <h2 class="module-header">Syllabus and schedule</h2>
    <table class="table">
        <th>Date</th><th>Type</th><th>Title</th><th>Speaker</th><th>Role</th><th>Affiliation</th><th>Presentation Date</th><th>Papers</th>
        <tr><td>9/5</td><td>Invited Speaker</td><td>Disease Phenotype Identification</td><td>Holger Hoefling</td><td>Lead of Scientific Data Analysis - Machine Learning and Quantative Analysis</td><td>Novartis</td><td></td><td><ul><li><a href="https://doi.org/10.1038/s41591-019-0508-1">Clinical-grade computational pathology using weakly supervised deep learning on whole slide images</a></li><li><a href="https://doi.org/10.1001/jama.2017.14585">Diagnostic Assessment of Deep Learning Algorithms for Detection of Lymph Node Metastases in Women With Breast Cancer</a></li><li><a href="https://doi.org/10.1016/j.media.2017.07.005">A survey on deep learning in medical image analysis</a></li></ul></td></tr><tr><td>9/10</td><td>Lecture</td><td>Overview of Target Identification</td><td>David Gifford</td><td>Professor</td><td>MIT</td><td></td><td></td></tr><tr><td>9/12</td><td>Invited Speaker</td><td>Systems Biology Based Target Identification</td><td>Ernest Frankel</td><td>Professor</td><td>MIT</td><td></td><td><ul><li><a href="https://www.sciencedirect.com/science/article/pii/S1535610818303581">Proteomics, Post-translational Modifications, and Integrative Analyses Reveal Molecular Heterogeneity within Medulloblastoma Subgroups</a></li><li><a href="https://www.nature.com/articles/nmeth.3940">Revealing disease-associated pathways by network integration of untargeted metabolomics</a></li><li><a href="https://www.nature.com/articles/ng.3371">The transcriptomic landscape and directed chemical interrogation of MLL-rearranged acute myeloid leukemias</a></li></ul></td></tr><tr><td>9/17</td><td>Presentation</td><td>Presentation on Computer-aided Drug Design (9/26 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>9/19</td><td>Presentation</td><td>Presentation on Small Molecule Design (10/8 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>9/24</td><td>Presentation</td><td>Presentation on Small Molecule Design (10/10 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>9/26</td><td>Invited Speaker</td><td>Using computational methods to address druggability and drug discovery challenges</td><td>Jose Duca</td><td>Global Head of Computer-Aided Drug Discovery</td><td>Novartis</td><td>9/17</td><td><ul><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/27362227">Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases</a></li><li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.8b00744">Using Membrane Partitioning Simulations To Predict Permeability of Forty-Nine Drug-Like Molecules</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/27595330">Small-molecule WNK inhibition regulates cardiovascular and renal function</a></li></ul></td></tr><tr><td>10/1</td><td>Presentation</td><td>Presentation (10/3 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>10/3</td><td>Invited Speaker</td><td>Introduction to drug development and the role of quantitative sciences‚Äù</td><td>Birgit Schoeberl</td><td>Global Head Modeling and Simulation, PK Sciences</td><td>Novartis Institutes for BioMedical Research</td><td>10/1</td><td><ul><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28649441">Clinical responses to ERK inhibition in BRAFV600E-mutant colorectal cancer predicted using a computational model.</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/30252552">Mathematical models of tumor cell proliferation: A review of the literature.</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/31119428">A Translational Quantitative Systems Pharmacology Model for CD3 Bispecific Molecules: Application to Quantify T Cell-Mediated Tumor Cell Killing by P-Cadherin LP DART</a></li><li><a href="https://www.nature.com/articles/s41540-017-0030-3">Predicting ligand-dependent tumors from multi-dimensional signaling features</a></li></ul></td></tr><tr><td>10/8</td><td>Invited Speaker</td><td>Junction Tree representations of small molecules</td><td>Tommi Jaakkola</td><td></td><td></td><td>9/19</td><td><ul><li><a href="https://arxiv.org/pdf/1802.04364.pdf">Junction Tree Variational Autoencoder for Molecular Graph Generation</a></li></ul></td></tr><tr><td>10/10</td><td>Invited Speaker</td><td>ML-driven small molecule selection in drug discovery</td><td>Jeremy Jenkins</td><td>Head of Data Science in Chemical Biology & Therapeutics,</td><td>Novartis</td><td>9/24</td><td><ul><li><a href="https://www.ncbi.nlm.nih.gov/m/pubmed/24933016">Using information from historical high-throughput screens to predict active compounds.</a></li><li><a href="https://www.ncbi.nlm.nih.gov/m/pubmed/16426055">Enrichment of high-throughput screening data with increasing levels of noise using support vector machines, recursive partitioning, and laplacian-modified naive bayesian classifiers.</a></li><li><a href="https://www.ncbi.nlm.nih.gov/m/pubmed/18066055">Integrating high-content screening and ligand-target prediction to identify mechanism of action.</a></li></ul></td></tr><tr><td>10/15</td><td>Holiday</td><td>Enjoy The Holiday!</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>10/17</td><td>Invited Speaker</td><td></td><td>Alex Zhavoronkov,</td><td>Founder and CEO</td><td>Insilico Medicine</td><td>9/19</td><td><ul><li><a href="https://www.nature.com/articles/s41587-019-0224-x">Deep learning enables rapid identification of potent DDR1 kinase inhibitors</a></li><li><a href="https://arxiv.org/abs/1811.12823">Molecular Sets, MOSES: A Benchmarking Platform for Molecular Generation Models</a></li><li><a href="https://www.nature.com/articles/ncomms13427">In silico Pathway Activation Network Decomposition Analysis (iPANDA) as a method for biomarker development</a></li></ul></td></tr><tr><td>10/22</td><td>Presentation</td><td>Syn Bio presentation (10/24 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>10/24</td><td>Invited Speaker</td><td>Synthetic Biology Approaches to Disease Therapeutics</td><td>Ron Weiss</td><td>Professor</td><td>MIT</td><td>10/22</td><td><ul><li><a href="https://www.nature.com/articles/ncomms10243">Genetically engineering self-organization of human pluripotent stem cells into a liver bud-like tissue using Gata6</a></li><li><a href="https://science.sciencemag.org/content/359/6376/eaad1067">Programming gene and engineered-cell therapies with synthetic biology</a></li><li><a href="https://science.sciencemag.org/content/333/6047/1307.full">Multi-Input RNAi-Based Logic Circuit for Identification of Specific Cancer Cells</a></li></ul></td></tr><tr><td>10/29</td><td>Presentation</td><td>Presentation on Vaccine Design (10/31 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>10/31</td><td>Invited Speaker</td><td>Design of Peptide Vaccines</td><td>Cathy Wu</td><td>Professor of Medicine</td><td>Dana-Farber Cancer Institute and Harvard Medical School</td><td>10/29</td><td><ul><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28678778">An Immunogenic Personal Neoantigen Vaccine for Patients with Melanoma</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28228285">Mass Spectrometry Profiling of HLA-Associated Peptidomes in Mono-allelic Cells Enables More Accurate Epitope Prediction. </a></li><li><a href="https://www.ncbi.nlm.nih.gov/pubmed/29226910">Towards personalized, tumour-specific, therapeutic vaccines for cancer.</a></li></ul></td></tr><tr><td>11/5</td><td>Presentation</td><td>Presentation on cell therapy (11/7 prep)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>11/7</td><td>Invited Speaker</td><td>Regenerative cell based therapies</td><td>Doug Melton</td><td>Professor</td><td>Harvard</td><td>11/5</td><td></td></tr><tr><td>11/12</td><td>Lecture</td><td>Machine Learning-based Antibody Design</td><td>David Gifford</td><td>Professor</td><td>MIT</td><td></td><td></td></tr><tr><td>11/14</td><td>Invited Speaker</td><td>CRISPR Therapeutic Strategies`</td><td>Han Altae-Tran</td><td>Graduate Student</td><td>Broad Institute</td><td></td><td></td></tr><tr><td>11/19</td><td>Invited Speaker</td><td>Cellular immune therapies</td><td>Michael Birnbaum</td><td>Professor</td><td>MIT</td><td>11/21</td><td></td></tr><tr><td>11/21</td><td>Presentation</td><td>Presentation on immune therapies (11/19)</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>11/26</td><td>No class</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>11/28</td><td>Holiday</td><td>Enjoy the Holiday!</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>12/3</td><td>Project Presentations</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>12/5</td><td>Project Presentations</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>12/10</td><td>Project Presentations</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
    </table>

    <h2 class="module-header">Prerequisites</h2>
    <p>Undergraduate version:  Fundamental knowledge of machine learning, programming, and biology (GIR level). You should be comfortable with calculus, linear algebra, (Python) programming, probability, and introductory molecular biology.   Graduate version: Understanding of machine learning with Python and commonly-used libraries.  The graduate version is targeted towards students with a high degree of fluency in Computation and Biology and to fully understand the material, students will be best off having previously taken machine learning or computational systems biolog (6.874) or a similar course.</p>
    
    <h2 class="module-header">Class meeting times and places</h2>
    <ul>
        <li>Lecture: TR 11AM-12:30PM MIT Room 56-114</li>
        <li>Lecture: T 2PM-3PM MIT Room 26-328</li>
    </ul>

    <h2 class="module-header">Contact</h2>
</p><p>The best way to get detailed questions answered is to visit TA office hours or post them on <a href="https://piazza.com/class/jzvee3jyo6z28i">Piazza</a>.</p>

    <h2 class="module-header">Office hours</h2>
    <div class="materials-item">
        David Gifford (gifford@mit.edu): Office hours by appointment
    </div>
    <div class="materials-item">
       Hyunjin Park (hyunjinp@mit.edu): TBD, Stata Center, G5 Lounge
    </div>

    <h2 class="module-header">Grading</h2>
    <p><p>Class presentations (30%), programming-intensive problem sets (30%), and a final project (40%). Attendance in lecture is important as the class moves quickly and you will need to be present. There will be a requirment for  students to give presentations on relevant research papers from the syllabus list. If you will need special accomodations, please email Student Support Services - S3). We will be happy to accomodate!</p>

    <h2 class="module-header">Project</h2>
    <p>Final project details TBA</p>

    <h2 class="module-header">Papers</h2>
    <p>Lectures will be given by MIT professors and guests from industry on current topics at the forefront of research in computational therapeutics design. These lectures will be accompanied by academic papers listed in the course schedule and available through the <a href="https://stellar.mit.edu/S/course/6/fa19/6.884/index.html"> Stellar site</a></p>
  </div>
</div>